sonar-plugin-api/src/main/java/org/sonar/api/issue/batch/IssueFilterChain.java

```diff
@@ -29,7 +29,9 @@
  *  <li>Let downstream filters decide by passing the issue to the rest of the chain</li>
  * </ul>
  * @since 4.0
+ * @deprecated since 5.3. Use {@link org.sonar.api.issue.filter.IssueFilterChain} instead.
  */
+@Deprecated
 public interface IssueFilterChain {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)